.editorconfig

```diff
@@ -161,6 +161,16 @@ dotnet_naming_symbols.interfaces_symbols.applicable_accessibilities = *
 dotnet_naming_style.I_pascal_case_style.required_prefix = I
 dotnet_naming_style.I_pascal_case_style.capitalization = pascal_case
 
+# name Local variables & parameters using camelCase
+dotnet_naming_rule.local_variables_and_parameters_camel_case.severity = error
+dotnet_naming_rule.local_variables_and_parameters_camel_case.symbols  = local_variables_and_parameters_symbols
+dotnet_naming_rule.local_variables_and_parameters_camel_case.style    = camel_case_style
+
+dotnet_naming_symbols.local_variables_and_parameters_symbols.applicable_kinds = local,parameter
+
+dotnet_naming_style.camel_case_style.capitalization = camel_case
+
```

Comment:
As far as I can understand this should not be allowed by the rule: 

`public async Task LoadOrCreateDefaultFileAsync(int MyParameter1, string MyPARAMETER2, int myParameter)`

But it is. Why? How is this working?

Reply:
@molnard It works fine for me.

Reply:
I did some local variable renaming: 
![image](https://user-images.githubusercontent.com/9844978/61416805-4afc1880-a8f5-11e9-9d4b-08c6c23be9ec.png)
No error or suggestions. It should underline it?